bears/js/ESLintBear.py

```diff
@@ -10,10 +10,13 @@
         use_stdin=True)
 class ESLintBear:
     """
-    Checks the code with ``eslint``. This will run eslint over each of the
-    files separately.
+    Check JavaScript and JSX code for style and semantic errors.
```

Comment:
i don't think ``style error`` is correct. maybe style issues and semantic errors?